be/src/vec/aggregate_functions/aggregate_function_window.h

```diff
@@ -231,6 +231,61 @@ class WindowFunctionPercentRank final
     void deserialize(AggregateDataPtr place, BufferReadable& buf, Arena*) const override {}
 };
 
+struct CumeDistData {
+    float64_t numerator = 0;
```

Comment:
It seems that you should include the particular header file to introduce `float64_t` for all architecture